### DISCUSSION TITLE
can be wrapped to a static or dynamic library for other language?

### DISCUSSION THREAD

**Discussion by flying1314** [pion/webrtc#1797] [Q&A] [answered]
if i want to use this project for other language, i think it should be wrapped to a static or dynamic library,is this can be done or not?if yes,please tell me what should i do to make it done?many thanks

**Comment by Sean-Der** [comment, marked as answer]
Yes this is being done today! See [c-data-channels](https://github.com/pion/example-webrtc-applications/tree/master/c-data-channels) this shows mixing C/Go code with Pion.
